prow/jobs/busola/busola-web/periodic-performance-tests-web.yaml

```
@@ -0,0 +1,15 @@
+# Periodic jobs
+
+# Description- This prow job is for running performance Tests on kyma web application. 
+periodics:
+- interval: 24h
+  agent: kubernetes
+  name: jmeter
```

The name of the the ProwJob is too general. Which repository does it rely to? If someone were to add similar periodic with that name they'd clearly run into name conflicts. Periodics need to have unique names because they are not bound to specific repo.

```suggestion
  name: ci-busola-web-jmeter
```